src/Context/AuthProvider/index.js

```diff
@@ -0,0 +1,21 @@
+import { createContext, useContext, useReducer } from "react";
+import { authReducer } from "../../Reducer/AuthReducer";
+import { loginUser, signUpUser } from "./AuthHandlers";
+
+const AuthContext = createContext();
+
+export function useAuthContext() {
+    return useContext(AuthContext);
+}
+
+export function AuthProvider({ children }) {
+    const [authState, authDispatch] = useReducer(authReducer, {
```

Comment:
prefer using camel case in reducer state or same for all states, you have used snake case for current user & camel case for logged in user